io.edgehog.devicemanager.apps.CreateContainerRequest.json

```diff
@@ -0,0 +1,97 @@
+{
+  "interface_name": "io.edgehog.devicemanager.apps.CreateContainerRequest",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "server",
+  "aggregation": "object",
+  "mappings": [
+    {
+      "endpoint": "/container/id",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 31556952,
+      "description": "Create Container Request id",
+      "doc": "Unique id for the container."
+    },
+    {
+      "endpoint": "/container/imageId",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 31556952,
+      "description": "Container image id",
+      "doc": "The id of the image to use then creating the container."
+    },
+    {
+      "endpoint": "/container/networkIds",
+      "type": "stringarray",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 31556952,
+      "description": "Container network ids",
+      "doc": "The ids of the network to connect with the container."
+    },
+    {
+      "endpoint": "/container/volumeIds",
+      "type": "stringarray",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 31556952,
+      "description": "Container volume ids",
+      "doc": "The ids of the volumes to mount on the container."
+    },
+    {
+      "endpoint": "/container/hostname",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 31556952,
+      "description": "Container hostname",
+      "doc": "The hostname to use for the container, as a valid RFC 1123 hostname."
+    },
+    {
+      "endpoint": "/container/restartPolicy",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 31556952,
+      "description": "Container restart policy",
+      "doc": "The behavior to apply when the container exits."
+    },
+    {
+      "endpoint": "/container/env",
+      "type": "stringarray",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 31556952,
+      "description": "Container environment",
+      "doc": "Array of KEY=value environment variables"
+    },
+    {
+      "endpoint": "/container/binds",
+      "type": "stringarray",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 31556952,
+      "description": "Container binds",
+      "doc": "Array of binds"
+    },
+    {
+      "endpoint": "/container/networks",
```

Comment:
What is the difference between this and `networkIds` above? Let's clarify it in the description

Reply:
I added:
- The id specifies a dependency
- The networks is a `strigarray` of the configuration to pass when creating the container

Reply:
The network field should better defined, for now in the device the network is passed as a key to an empty object in the container's NetworkConfig